### PR TITLE
Working syndici field on building page.

### LINKED ISSUE #361: "Eigenaar" is not correctly shown in building add page
image

### PR DESCRIPTION
**Pull request by milachae**
Closes #361 

Syndici field on the building page should work now.

### CODE REVIEW

File: frontend/src/components/forms/BuildingForm.jsx [hunk 1]
```diff
@@ -48,13 +50,21 @@ export default function BuildingForm({ id }) {
     }
 
     try {
+      if (owner === "") {
+        alert("No owner given.");
+        return;
+      }
+
```
Comment (JasperJanin) [inline frontend/src/components/forms/BuildingForm.jsx:57]: Are we requiring an owner?

Comment (milachae) [reply frontend/src/components/forms/BuildingForm.jsx:57]: I don't know. Are we @eliasnijs

Comment (JasperJanin) [reply frontend/src/components/forms/BuildingForm.jsx:57]: Let's leave freedom to the users here

Comment (milachae) [reply frontend/src/components/forms/BuildingForm.jsx:57]: Ok, I will change it!

**Review by renevds — APPROVED** [review]
GJ

File: frontend/src/components/forms/BuildingForm.jsx (hunk 1 above)
Comment (milachae) [reply frontend/src/components/forms/BuildingForm.jsx]: @JasperJanin this should be fixed now

**Review by JasperJanin — APPROVED** [review]

**Pull request merged**